.changes/config.json

```diff
@@ -212,16 +203,16 @@
         "tauri-runtime-wry",
         "tauri-build"
       ],
-      "postversion": "node ../../.scripts/ci/sync-cli-metadata.js ${ pkg.pkg } ${ release.type }"
+      "postversion": [
+        "node ../../.scripts/ci/sync-cli-metadata.js ${ pkg.pkg } ${ release.type }",
+        "cargo build --manifest-path ../tauri-schema-generator/Cargo.toml"
```

Comment:
should probably do this only when tauri-utils bumps instead

Reply:
nvm when utils bump tauri bumps too anyway.. and the schema is tied to the tauri version